subsys/net/lib/fota_download/src/util/fota_download_util.c

```diff
@@ -32,6 +32,8 @@
 #include "fota_download_smp.h"
 #endif
 
+#include "../../download_client/src/download_client_internal.h"
```

Comment:
Can CMake help finding this include instead of moving up the tree?

Reply:
I changed this to:
```
zephyr_include_directories_ifdef(CONFIG_DOWNLOAD_CLIENT
  ${ZEPHYR_NRF_MODULE_DIR}/subsys/net/lib/download_client/include)
```

Is that better?

Reply:
Yes, I think this looks better.